.github/workflows/speakeasy_sdk_generation.yml

```diff
@@ -1,29 +1,25 @@
 name: Generate
 permissions:
-  checks: write
-  contents: write
-  pull-requests: write
-  statuses: write
+    checks: write
+    contents: write
+    pull-requests: write
+    statuses: write
 "on":
-  workflow_dispatch:
-    inputs:
-      force:
-        description: Force generation of SDKs
-        type: boolean
-        default: false
-  schedule:
-    - cron: 0 0 * * *
+    workflow_dispatch:
+        inputs:
+            force:
+                description: Force generation of SDKs
+                type: boolean
+                default: false
+    schedule:
+        - cron: 0 0 * * *
 jobs:
-  generate:
-    uses: speakeasy-api/sdk-generation-action/.github/workflows/sdk-generation.yaml@v14
-    with:
-      force: ${{ github.event.inputs.force }}
-      languages: |
-        - go
-      mode: direct
-      openapi_docs: |
-        - https://api-docs.bolt.com/api.bundle.yaml
-      speakeasy_version: 1.210.0
```

Comment:
unpinned old generation version. You can always keep this pin if you'd like too, but we recommend staying up to date.